css/style.css

```diff
@@ -2018,3 +2018,84 @@ the home page, intro pitch and your-code-here layed out vertically */
 dt.d_decl:hover .decl_anchor {
     visibility: visible;
 }
+
+/* Friends of D */
+.orgs-using-d {
+    width: 100%;
+    padding-left: 20px;
+}
+
+.orgs-using-d-cell {
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
`auto` is the default value for `margin-left` and `margin-right`. No need to specify it here, as far as I see.